$MirOS: ports/print/cups/patches/patch-berkeley_lprm_c,v 1.1.7.1 2005/03/18 15:51:00 tg Exp $
--- berkeley/lprm.c.orig	Sun Jan 29 14:39:21 2006
+++ berkeley/lprm.c	Tue May  9 17:39:25 2006
@@ -232,7 +232,7 @@ main(int  argc,			/* I - Number of comma
       }
       else
       {
-        sprintf(uri, "ipp://localhost/jobs/%d", job_id);
+        snprintf(uri, sizeof(uri), "ipp://localhost/jobs/%d", job_id);
 	ippAddString(request, IPP_TAG_OPERATION, IPP_TAG_URI, "job-uri", NULL,
 	             uri);
       }
